src/main/sensors/acceleration.c

```diff
@@ -92,6 +92,7 @@ extern bool AccInflightCalibrationActive;
 static flightDynamicsTrims_t *accelerationTrims;
 
 static biquadFilter_t accFilter[XYZ_AXIS_COUNT];
+static bool accFilterInitialised = false;
```

Comment:
`accFilterInitialised` is no longer required, now that `accSamplingInterval` is set.